templates/home/Features/features.html

```diff
@@ -1,140 +1,52 @@
-    <title>www.codingeasy.com</title>
+<head>
 
-  </head>
-  <body>
-    <!-- Preloader -->
-    <!-- <div id="preloader">
-      <div id="loader"></div>
-      </div> -->
-    <!-- /Preloader  -->
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-      <div class="container-fluid">
-        <a class="navbar-brand" href="#">
-          <i class="fas fa-hands-helping"></i>
-        </a>
-        <a class="navbar-brand" href="index.html">Coding Easy</a>
-        <button
-          class="navbar-toggler"
-          type="button"
-          data-bs-toggle="collapse"
-          data-bs-target="#navbarNavDropdown"
-          aria-controls="navbarNavDropdown"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
-        >
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse" id="navbarNavDropdown">
-          <ul class="navbar-nav">
-            <li class="nav-item">
-              <a class="nav-link active" aria-current="page" href="{% url 'index' %}"
-                >Home</a
-              >
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="/feature">Features</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="/pricing">Pricing</a>
-            </li>
-            <li class="nav-item">
-              <a class="text-md-end nav-link" href="/login">Login/Signup</a>
-            </li>
-            <li class="nav-item">
-              <a class="text-md-end nav-link" href="/contact">Contact Us</a>
-            </li>
-          </ul>
-        </div>
-      </div>
-    </nav>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="{% static 'css/common.css' %}">
+  <link rel="stylesheet" href="{% static 'css/features/features.css' %}">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" rel="stylesheet">
+  <title>Features | Coding Easy</title>
 
-      <br>
-       <div class="heading text-center display-3">
-           CodingEasy
-       </div> <br>
+</head>
 
-      <br><br>
+<body>
 
-      <h2 class="fea">      In-depth concept coverage</h2>
-      <h2  class="fea">      One step solution</h2>
-      <h2  class="fea">      24*7 Doubt solving</h2>
-      <h2  class="fea">      1 on 1 Mentoring</h2>
-      <h2  class="fea">      Project Guidance</h2>
-      <h2  class="fea">      Certification Included</h2>
-      <br><br>
+  {% include 'includes/navbar.html' %}
```

Comment:
This include is not required. Inherit directly from base.html. For reference [base.html](https://github.com/arpit456jain/CodingEasy/blob/master/templates/home/base.html) and [button.html](https://github.com/arpit456jain/CodingEasy/blob/master/templates/home/bootstrap/button/button.html). Including navbar.html in every template does not make any sense. The same goes for footer.

Reply:
Alright. Will try to resolve the matter right away.